servicelog/scraper/json.go

```diff
@@ -71,7 +80,7 @@ func (j *JSON) scanLoop(reader io.Reader, logEntries chan<- servicelog.Entry) er
 			}
 		}
 		if j.BufferSize > 0 && len(logEntries) >= int(j.BufferSize) {
-			log.Warnf("Dropping logs because of a buffer overflow (buffer size %s)", j.BufferSize)
```

Comment:
not everyone is looking at metrics, everyone from time to time is looking at logs :)

Reply:
Seems redundant and in case of overflow, the higher volume of this could only obfuscate log, metric is more suited for the task.